cmd/beacon-chain/flags/base.go

```diff
@@ -262,8 +262,9 @@ var (
 		Value: cmd.DefaultDataDir(),
 	}
 	BlobRetentionEpoch = &cli.Uint64Flag{
-		Name:  "extend-blob-retention-epoch",
-		Usage: "Extend blob retention epoch period to beyond default 4096 epochs (~18 days). The node will error at start if input value is less than 4096 epochs.",
-		Value: uint64(params.BeaconNetworkConfig().MinEpochsForBlobsSidecarsRequest),
+		Name:    "blob-retention-epoch",
+		Usage:   "Extend blob retention epoch period to beyond default 4096 epochs (~18 days). The node will error at start if input value is less than 4096 epochs.",
+		Value:   uint64(params.BeaconNetworkConfig().MinEpochsForBlobsSidecarsRequest),
+		Aliases: []string{"extend-blob-retention-epoch"},
```

Comment:
Minor changes. I think it should be plural and the usage amended to match the name. 

```suggestion
		Name:    "blob-retention-epochs",
		Usage:   "Set the blob retention epoch period. The default is 4096 epochs (~18 days) and the node will error at start if input value is less than 4096 epochs.",
		Value:   uint64(params.BeaconNetworkConfig().MinEpochsForBlobsSidecarsRequest),
		Aliases: []string{"extend-blob-retention-epoch"},
```